content/tutorials/first-aws-environment.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Your first environment on AWS"
+description: "Get your first AWS environment deployed using Atmos, Stacks, and Vendir"
+weight: 3
+---
+
+## Intro
+
+Up until now, our tutorial content has been been primarily using contrived examples to enable folks to get up to speed with some of the more fundamental SweetOps concepts and tools. This tutorial will be a bit different in that we'll actually get something deployed to your AWS account so you can see how SweetOps works in a more concrete way.
+
+## Prerequisites
+
+### System Requirements
+
+To accomplish this tutorial, you'll need the following:
+
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [Docker](https://docs.docker.com/get-docker/)
+- An AWS test account and credentials for that account added to `aws-vault` so they're ready to be used in Geodesic.
+  - We recommend Administrator privileges to make everything easy, but you can likely get away with credentials that only allow S3, ClodFront, and DynamoDB access.
+  - If you haven't set this up before, check out [the Geodesic how-to on authenticating with aws-vault]({{< relref "howto/geodesic/authenticate-with-aws-vault.md" >}}).
+  - We'll refer to your `aws-vault` profile through out this tutorial as `$YOUR_PROFILE`. When you see that, be sure to replace with whatever name you added to `aws-vault`.
+
+### Understanding
+
+Prior to starting this tutorial, you should be sure you understand [our various concepts and terminology]({{< relref "fundamentals/concepts.md" >}}) and have gone through our [Getting started with Geodesic]({{< relref "tutorials/geodesic-getting-started.md" >}}) and [Getting started with Atmos tutorial]({{< relref "tutorials/atmos-getting-started.md" >}}) because we'll be building upon the knowledge in both of those tutorials.
```

Comment:
```suggestion
Prior to starting this tutorial, you should be sure that you understand [our various concepts and terminology]({{< relref "fundamentals/concepts.md" >}}) and have gone through our [Getting started with Geodesic]({{< relref "tutorials/geodesic-getting-started.md" >}}) and [Getting started with Atmos tutorial]({{< relref "tutorials/atmos-getting-started.md" >}}) because we'll be building upon the knowledge in both of those tutorials.
```